### DISCUSSION TITLE
did not find expected tag URI, why it happens?

### DISCUSSION THREAD

**Discussion by MohamedXi** [dunglas/symfony-docker#175] [Q&A] [answered]
I encounter an error that I can't solve.
Have any of you encountered this error?

> yaml: line 9: did not find expected tag URI

I have not changed anything.

**Comment by rowanparker** [comment, marked as answer]
This happened to me too. It was a Docker issue.
Go into Settings > Experimental Features and disable Use Docker Compose V2 release candidate.
